docs/user-guide/prerequisite.md

```diff
@@ -0,0 +1,27 @@
+###...
+
+# Prerequisites
+## VMWare Environment - vSphere/Photon
+
+You will need an environment with VMWare vSphere or Photon capabilities. Additionally since this plugin is focused on storage, you will need one of the storage technologies mentioned in a later section.
+
+## Docker
+On all the machines where you want to operate the vDVS, you will need docker-engine installed. You can get the installable specific to your OS form Docker website
+
+## Docker volume plugin
```

Comment:
We should mention two components that vDVS is composed of -

1. Managed Plugin available on [Docker Store](https://store.docker.com/plugins/vsphere-docker-volume-service?tab=description)
2. VIB - Available on [Bintray](https://bintray.com/vmware/vDVS)